packages/base/src/config/Icons.ts

```diff
@@ -88,6 +88,7 @@ const getEffectiveIconCollection = (collectionName?: IconCollection): IconCollec
  * - "tnt" (and its alias "SAP-icons-TNT") resolves to "tnt-v2" in "Quartz", "Belize", and resolves to "tnt-v3" in "Horizon"
  * - "tnt-v2" forces "TNT icons v2" in any theme and resolves to itself "tnt-v2"
  * - "tnt-v3" forces "TNT icons v3" in any theme and resolves to itself "tnt-v3"
+ * - "business-suite" (and its alias "SAP-icons-business-suite") resolves to "business-suite-v1" in "Quartz", "Belize", and resolves to "business-suite-v2" in "Horizon"
```

Comment:
The alias actually is "BusinessSuiteInAppSymbols", you can try BusinessSuiteInAppSymbols/3d (add):

These should result in the same icon:
```html
<ui5-icon class="samples-margin" name="business-suite/add-polygone"></ui5-icon>
<ui5-icon class="samples-margin" name="BusinessSuiteInAppSymbols/add-polygone"></ui5-icon>
```